CONTRIBUTING.md

```diff
@@ -105,6 +109,18 @@ We use [CSS Modules](https://github.com/css-modules/css-modules) along with [BEM
 
 When creating (S)CSS files, follow the CSS Module naming convention by using the `.module.(s)css` extension.
 
+When creating or modifying SCSS files, follow these rules
+
+1. Use Modern SASS API
+   * Prefer `@use` instead of `@import`
+   * Prefer `math.div($a, $b)` instead of `$a / $b`. Add `@use sass:math` statement to the top of your file to make this function available
+   * Read more about [@use rule](https://sass-lang.com/documentation/at-rules/use/) and [SASS math functions](https://sass-lang.com/documentation/modules/math/)
+2. Use only what you need
+   * Instead of blank import of all mixins, import them on demand. E.g. if you need only colour tokens, add `@use '../unstable_bpk-mixins/bonds'` statement only
```

Comment:
what does the word "bond" mean in this context?

Reply:
That's the name of the file re-exporting tokens. Maybe @olliecurtis knows more about the historic reasons for this naming?

Reply:
This naming sadly predates me at Skyscanner and that original naming knowledge has left Skyscanner. Never really questioned it in the past and just left it as is as there was maybe a reason to it

Reply:
Something to think about then as this is now going from an internal API, to an external one proliferated across all microsites.

We either need a reason why "bond" is used, or think about restructuring so that the files are intuitive for consumers to use

Reply:
Maybe just rename to tokens?